File: 1-Iniciante/JavaScript/1044.js
let [a, b] = lines.shift().split(' ').map((x) => parseInt(x));

if(a < b)   [a, b] = [b, a];

console.log((a % b === 0) ? "Sao Multiplos" : "Nao sao Multiplos");

// usando resta zero % e operador condicional ?

File: 1-Iniciante/Java/1044.java
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args){
 
        Scanner in = new Scanner(System.in);
	    int num1 = in.nextInt();
	    int num2 = in.nextInt();
	    
	    if (num1%num2 == 0) System.out.println("Sao Multiplos");
	    else if (num2%num1 == 0) System.out.println("Sao Multiplos");
	    else System.out.println("Nao sao Multiplos");
 
    }
 
}
